indexer/bag_indexer/src/main/resources/application.conf

```diff
@@ -1,5 +1,8 @@
 aws.metrics.namespace=${?metrics_namespace}
 aws.sqs.queue.url=${?queue_url}
+aws.vhs.dynamo.tableName=${?vhs_table_name}
+aws.vhs.s3.bucketName=${?vhs_bucket_name}
+aws.vhs.s3.globalPrefix=${?vhs_bucket_name}
```

Comment:
The fact that you need these flags to do a read-only VHS is a bit of an antipattern. Not one to fix here, but one of the things that makes multiple apps accessing the VHS a pain.